src/CMakeLists.txt

```diff
@@ -109,13 +109,11 @@ endif(NOT APPLE)
 
 target_link_libraries(mosquitto ${MOSQ_LIBS})
 
-if (UNIX)
```

Comment:
I believe this simplification is incorrect because it will break the build on Windows.